packages/ui-extensions-server-kit/src/types.ts

```diff
@@ -103,3 +103,7 @@ export interface App {
   supportEmail?: string
   supportLocales?: string[]
 }
+
+interface ExtensionTranslationMap {
+  [key: string]: string
```

Comment:
Is it possible to have nested objects or are translations always a flat structure ?

Reply:
Ah, the `app/foundation/i18n/en.json` file is just being used for the Hub UI translations and not for the extensions. For  `customer-account-ui-extensions` we won't have nested objects translations for now. 

Thanks for your comment :)